### ISSUE TITLE
[Garage] It will be done?

### ISSUE THREAD

**Issue by Dokman** [Madriax/Apartments#4]
How the title said, the garage will be done? this is the question :(

**Comment by Madriax** [comment]
I will work on that next week. First I have to finish all my final exams

**Comment by loko06320** [comment]
i do it , juste add on your es_garages\client.lua


`	------------------------------Appartements-------------------------------
	{name="Garage Privé", colour=79, id=357, x=353.688720313, y=436.97164916992, z=145.500},-- apartement (saloo) house prenium 4
	{name="Garage Privé", colour=79, id=357, x=-7.0757069587708, y=-574.12805175781, z=36.500},-- apartement condo 3 + 4 
	{name="Garage Privé", colour=79, id=357, x=-795.999633378906, y=324.50518798828, z=84.500},-- apartement eclipse tower
	{name="Garage Privé", colour=79, id=357, x=-892.39984130859, y=-338.30395507813, z=33.500},-- appartement 21/20/19
	{name="Garage Privé", colour=79, id=357, x=-834.00286865234, y=-393.78768920898, z=30.500},-- apartement 22/23
	{name="Garage Privé", colour=79, id=357, x=-1380.1951904297, y=-475.09963989258, z=30.500},-- apartement 13/14/17/18
	{name="Garage Privé", colour=79, id=357, x=-622.16394042969, y=56.282577514648, z=42.500},-- apartement 10
	{name="Garage Privé", colour=79, id=357, x=-188.4751739502, y=501.71194458008, z=133.000},-- apartement 8
	{name="Garage Privé", colour=79, id=357, x=131.34097290039, y=567.98199462891, z=182.200},-- apartement 6
	{name="Garage Privé", colour=79, id=357, x=-292.90454101563, y=-990.73059082031, z=23.000},-- apartement 1
	{name="Garage Privé", colour=79, id=357, x=-472.51666259766, y=-727.49114990234, z=29.500},-- apartement 4
	{name="Garage Privé", colour=79, id=357, x=-3089.1120605469, y=340.9645690918, z=6.500},-- apartement 25`

**Comment by b4b4s666** [comment]
That gonna add private garage for each buyable house?? If yes wow great job
my friend

On Jul 5, 2017 11:16 PM, "loko06320" <notifications@github.com> wrote:

i do it , juste add on your es_garages\client.lua

------------------------------Appartements-------------------------------
{name="Garage Privé", colour=79, id=357, x=353.688720313,
y=436.97164916992, z=145.500},-- apartement (saloo) house prenium 4
{name="Garage Privé", colour=79, id=357, x=-7.0757069587708,
y=-574.12805175781, z=36.500},-- apartement condo 3 + 4 {name="Garage
Privé", colour=79, id=357, x=-795.999633378906, y=324.50518798828,
z=84.500},-- apartement eclipse tower {name="Garage Privé", colour=79,
id=357, x=-892.39984130859, y=-338.30395507813, z=33.500},-- appartement
21/20/19 {name="Garage Privé", colour=79, id=357, x=-834.00286865234,
y=-393.78768920898, z=30.500},-- apartement 22/23 {name="Garage Privé",
colour=79, id=357, x=-1380.1951904297, y=-475.09963989258, z=30.500},--
apartement 13/14/17/18 {name="Garage Privé", colour=79, id=357,
x=-622.16394042969, y=56.282577514648, z=42.500},-- apartement 10
{name="Garage Privé", colour=79, id=357, x=-188.4751739502,
y=501.71194458008, z=133.000},-- apartement 8 {name="Garage Privé",
colour=79, id=357, x=131.34097290039, y=567.98199462891, z=182.200},--
apartement 6 {name="Garage Privé", colour=79, id=357, x=-292.90454101563,
y=-990.73059082031, z=23.000},-- apartement 1 {name="Garage Privé",
colour=79, id=357, x=-472.51666259766, y=-727.49114990234, z=29.500},--
apartement 4 {name="Garage Privé", colour=79, id=357, x=-3089.1120605469,
y=340.9645690918, z=6.500},-- apartement 25

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Madriax/Apartments/issues/4#issuecomment-313284216>,
or mute
the thread

**Comment by loko06320** [comment]
yes , but is not a garages like gta online , but es_garage near the appart ...

**Comment by b4b4s666** [comment]
thx loko ill test it right now !!

**Comment by b4b4s666** [comment]
work perfectly
thanks

**Comment by Madriax** [comment]
I'll add the ability to buy private garage, then you'll just have to adapt it to your main garage script

**Comment by FlamingOcean** [comment]
> I'll add the ability to buy private garage, then you'll just have to adapt it to your main garage script

have you done it?

**Comment by Leafgard** [comment]
Hey, I wanted to know if this is still up to date ?! xD
